src/main/java/io/github/mzmine/parameters/parametertypes/OptionalEditor.java

```diff
@@ -58,6 +59,11 @@ public OptionalEditor(PropertySheet.Item parameter) {
                     .getDeclaredConstructor(PropertySheet.Item.class)
                     .newInstance(embeddedParameter);
             Node embeddedNode = embeddedEditor.getEditor();
+            embeddedNode.setDisable(true);
```

Comment:
This causes all optional parameter fields to be disabled when opening a parameter dialog - not accounting for those checkboxes which are selected.